app/femr/common/dtos/ServiceResponse.java

```diff
@@ -18,10 +18,12 @@
 */
 package femr.common.dtos;
 
+import femr.common.models.MissionTripItem;
+
 import java.util.HashMap;
 import java.util.Map;
 
-public class ServiceResponse<T> {
+public class ServiceResponse<T> extends MissionTripItem {
```

Comment:
I'm not entirely sure how this got here. It definitely is not necessary, it should be:

`public class ServiceResponse<T> {`